proto/edgehog/deviceruntime/containers/v1/v1.proto

```diff
@@ -57,46 +49,52 @@ message GetResponse {
 
 // Request to start a container.
 message StartRequest {
-  // The ID of the container to start.
+  // The Edgehog ID of the container to start.
   string id = 1;
 }
 
 // Request to stop a container.
 message StopRequest {
-  // The ID of the container to stop.
+  // The Edgehog ID of the container to stop.
   string id = 1;
   // The timeout in seconds to wait for the container to stop.
   optional int32 timeout_seconds = 2;
 }
 
 // Request to stream container statistics.
 message StatsRequest {
-  // The ID of the container to get statistics for.
-  repeated string id = 1;
+  // The Edgehog ID of the containers to get statistics for.
+  //
+  // If no ID is passed, all container telemetry is sent.
+  repeated string ids = 1;
   // Interval in seconds between stats updates.
   optional int32 interval_seconds = 2;
+  // Limit the number of values to receive.
+  optional uint64 limit = 3;
 }
 
 // A single stream of container statistics.
 message StatsResponse {
-  // ID of the container
-  string container_id = 1;
+  // Edgehog ID of the container
+  string id = 1;
+  // Container ID from the container runtime.
+  optional string container_id = 2;
   // Date and time at which this sample was collected
-  google.protobuf.Timestamp read = 2;
-  // Date and time at which this first sample was collected
-  google.protobuf.Timestamp prepared = 3;
+  google.protobuf.Timestamp read = 3;
+  // Date and time at which the previous sample was collected
+  optional google.protobuf.Timestamp preread = 4;
   // Stats of a container's process-IDs (PIDs).
-  optional PidsStats pids_stas = 4;
+  optional PidsStats pids_stats = 5;
   // BlkioStats stores all IO service stats for data read and write
-  optional BlkioStats blkio_stats = 5;
+  optional BlkioStats blkio_stats = 6;
   // CPU related info of the container
-  optional CpuStats cpu_stats = 6;
+  optional CpuStats cpu_stats = 7;
   // CPU related info of the container
-  optional CpuStats precpu_stats = 7;
+  optional CpuStats precpu_stats = 8;
```

Comment:
Could you improve the comment on this field? I'm having issues understanding what this is.